netbox/tenancy/tables/columns.py

```diff
@@ -24,3 +26,32 @@ def __init__(self, *args, **kwargs):
 
     def value(self, value):
         return str(value) if value else None
+
+
+class TenantGroupColumn(tables.TemplateColumn):
+    """
+    Include the tenant group description.
+    """
+    template_code = """
+    {% if record.tenant and record.tenant.group %}
```

Comment:
This is where we need to be careful with regard to prefetching. Because the column itself doesn't define an accessor, the tenant group doesn't get prefetched in the queryset. Instead, a separate query is made for `tenant.group` as needed for each unique tenant. I think we can resolve this by specifying `tenant__group` as the accessor for the column and tweaking the template code, but I'll merge this as-is and try a few things.